src/ui.ts

```diff
@@ -154,7 +154,7 @@ export const DEFAULT_VIEW_STATE_BY_PRIVACY_REGIME: PrivacyRegimeToInitialViewSta
     // US: Colorado
     CPA: InitialViewState.Hidden,
     // US: Nevada
-    'Nevada SB220': InitialViewState.Hidden,
+    [PrivacyRegimeEnum.NEVADA_SB220]: InitialViewState.Hidden,
```

Comment:
Shouldn't it just be 
```suggestion
    NEVADA_SB220: InitialViewState.Hidden,
```
like the others?